pkg/model/serializer.go

```diff
@@ -286,7 +288,12 @@ func (buildInfo *BuildInfo) UnmarshalYAML(unmarshal func(interface{}) error) err
 	var rawString string
 	err := unmarshal(&rawString)
 	if err == nil {
+		if strings.HasPrefix(rawString, "./") || strings.HasPrefix(rawString, ".") {
+			buildInfo.Context = rawString
+			return nil
+		}
 		buildInfo.Name = rawString
```

Comment:
@jLopezbarb why is this? in this case, only `context` is defined, no?

Reply:
i added this because rawString is assigned as Name when using BuildInfo, but in the build manifest, if the build is a string can be the rel path to the dockerfile...